src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -175,7 +175,7 @@ public async Task<CompletionSet> GetLibraryCompletionSetAsync(string libraryName
             try
             {
                 // library name completion
-                if (caretPosition < name.Length + 1)
+                if (caretPosition < name.Length + 1 || string.IsNullOrEmpty(name) && libraryNameStart.StartsWith("@", StringComparison.Ordinal))
```

Comment:
I'd Trim() libraryNameStart at the beginning of this method to make this code a bit more resilient to possible leading or trailing spaces.

Reply:
I don't think this is the right fix.  We're getting the wrong data back from the LibraryIdToNameAndVersionConverter (which in this case is using VersionedLibraryNamingScheme) when a scoped package doesn't have a 2nd @ to delineate the version number.